File: Code Chef/JavaScript/500-599/591 Chef and Bird farm BIRDFARM.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.trim().split(' ').map(Number));
    
    return intArray;
};

const checkBirdType = (chickenLegs, duckLegs, totalLegs) => {
    const isChicken = (totalLegs % chickenLegs) === 0;
    const isDuck = (totalLegs % duckLegs) === 0;
    
    (isChicken && isDuck)
        ? console.log('ANY')
        : (!isChicken && !isDuck)
            ? console.log('NONE')
            : (isChicken && !isDuck)
                ? console.log('CHICKEN')
                : console.log('DUCK');
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((testcase) => {
        const [chickenLegs, duckLegs, totalLegs] = testcase;
        checkBirdType(chickenLegs, duckLegs, totalLegs);
    });
});

File: Code Chef/C Programming/500-599/591 Chef and Bird farm BIRDFARM.c
#include <stdio.h>
int main() 
{
    int t,x,y,z;
    scanf("%d",&t);
    for(int i=0;i<t;i++)
    {
        scanf("%d%d%d",&x,&y,&z);
        if(((z%x)==0)&&((z%y)==0))
        {
            printf("ANY\n");
        }
        else if(((z%x)!=0)&&((z%y)!=0))
        {
            printf("NONE\n");
        }
        else if(((z%x)==0)&&((z%y)!=0))
        {
            printf("CHICKEN\n");
        }
        else if(((z%x)!=0)&&((z%y)==0))
        {
            printf("DUCK\n");
        }
    }
	return 0;
}